spec/numberformat.html

```diff
@@ -457,16 +540,33 @@
       <emu-alg>
         1. Let _nf_ be *this* value.
         1. If Type(_nf_) is not Object, throw a *TypeError* exception.
-        1. If _nf_ does not have an [[initializedNumberFormat]] internal slot, throw a *TypeError* exception.
-        1. If _nf_.[[boundFormat]] is *undefined*, then
+        1. If _nf_ does not have an [[initializedNumberFormat]] internal slot whose value is *true*, throw a *TypeError* exception.
+        1. If the [[boundFormat]] internal slot of _nf_ is *undefined*, then
           1. Let _F_ be a new built-in function object as defined in Number Format Functions (<emu-xref href="#sec-number-format-functions"></emu-xref>).
           1. Let _bf_ be BoundFunctionCreate(_F_, _nf_, « »).
           1. Perform ! DefinePropertyOrThrow(_bf_, `"length"`, PropertyDescriptor {[[Value]]: 1, [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *true*}).
-          1. Set _nf_.[[boundFormat]] to bf.
+          1. Set _nf_.[[boundFormat]] to _bf_.
         1. Return _nf_.[[boundFormat]].
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-intl.numberformat.prototype.formattoparts">
+      <h1>Intl.NumberFormat.prototype.formatToParts ([ value ])</h1>
```

Comment:
@caridy This wouldn't change any logic -- it would simply change the `.length` of the function, which wouldn't be consulted by anything using `.format` directly (because if you know the identity of your callee that way, you wouldn't have any reason to check its `.length`).  If you wanted to be more clear that there's no semantic change in the function's behavior, and that the only change is to the value of its `.length`, you could have the exact same effect by adding, "The value of the **`length`** property of the **`formatToParts`** method is 1." just as `Intl.Collator.supportedLocalesOf` has such language.